eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -311,6 +322,13 @@ jobs:
         sbomEnabled: false
 
   steps:
+  - ${{ if not(parameters.isBuiltFromVmr) }}:
+    # Synchronize new content in the VMR during PRs (we expect this to come

```

Comment:
This comment doesn't seem complete.

Reply:
Yeah, it was the original comment I brought back along with the surrounding code. I'll remove the content in parenthesis as it doesn't add any value.

Here's the same comment in P4 branch: https://github.com/dotnet/dotnet/blob/c22dcd0c7a78d095a94d20e59ec0271b9924c82c/src/sdk/eng/pipelines/templates/jobs/vmr-build.yml#L339

Reply:
Fixes with https://github.com/dotnet/dotnet/pull/548/commits/9981a4f32fd0601604d18d166fa325a2bfaba45a